oauthlib/oauth1/rfc5849/signature.py

```diff
@@ -334,75 +313,17 @@ def collect_parameters(uri_query='', body=[], headers=None,
     return unescaped_params
 
 
-def normalize_parameters(params):
-    """**Parameters Normalization**
-    Per `section 3.4.1.3.2`_ of the spec.
-
-    For example, the list of parameters from the previous section would
-    be normalized as follows:
-
-    Encoded::
-
-    +------------------------+------------------+
```

Comment:
Why did you remove the example?

Reply:
It added little value, because:

The reader can easily load up the RFC and read it directly.
The inline comments for the algorithm are already present.
I doubt readers will scroll up and down between the code and example. And if they did, it would be easier done in a separate window (point 1).
If the whole example was to be added as a comment, it would make more sense to copy it into the unit test tests/oauth1/rfc5849/test_signatures.py which uses the example for its testing. But again, duplicating the RFC verbatim doesn't add much value.